### PR TITLE
Pedrobn23/documentation fixed

### PR DESCRIPTION
**Pull request by pedrobn23**
Improve documentation.

### CODE REVIEW

File: python/pyglome/glome.py [hunk 1]
```diff
@@ -114,7 +114,8 @@ def __init__(self,
               Must be an integer in range 1-32.
         Raises:
             ValueError: Raised whenever min_peer_tag_len is not in
-              range 1-32.
+              range 1-32. Also when null shared secret is derived from
+              user/peer key pair.
```
Comment (bluecmd) [inline python/pyglome/glome.py:118]: When does that happen? If it is the .exchange() function that returns null, consider using a specific error such as ExchangeError or something and actively check for None.

**Review by bluecmd — CHANGES_REQUESTED** [review]

File: python/pyglome/glome.py (hunk 1 above)
Comment (pedrobn23) [reply python/pyglome/glome.py:118]: Is the exchange function who raises ValueError exception when some public keys are provided. WDYT about `raise ExchangeError from ValueError`?

File: python/pyglome/glome.py [hunk 2]
```diff
@@ -248,7 +257,8 @@ def __init__(self,
              documentation.
         Raises:
            ValueError: Raised whenever min_peer_tag_len is not in range 1-32 or
-             skippable_length is a negative integer.
+             skippable_length is a negative integer. Also when null shared secret
```
Comment (bluecmd) [inline python/pyglome/glome.py:260]: ExchangeError

**Review by bluecmd — CHANGES_REQUESTED** [review]

File: python/pyglome/glome.py (hunk 2 above)
Comment (pedrobn23) [reply python/pyglome/glome.py]: Fixed.

**Review by bluecmd — APPROVED** [review]

**Pull request merged**